lib/federation/docker-compose.federation.yaml

```diff
@@ -3,13 +3,6 @@ version: '3.7'
 services:
 # -- base federation service
   federation:
-    build:
-      context: $PWD/lib/federation
-      args:
-        BASE_IMAGE: ${BENTOV2_FEDERATION_BASE_IMAGE}
-        BASE_IMAGE_VERSION: ${BENTOV2_FEDERATION_BASE_IMAGE_VERSION}
-        REPO: ${BENTO_FEDERATION_REPO}
-        TAG_OR_BRANCH: ${BENTOV2_FEDERATION_TAG_OR_BRANCH}
     image: ${BENTOV2_FEDERATION_IMAGE}:${BENTOV2_FEDERATION_VERSION}
     container_name: ${BENTOV2_FEDERATION_CONTAINER_NAME}
     environment: 
```

Comment:
`CHORD_REGISTRY_URL`, `BENTO_FEDERATION_MODE`, `INITIALIZE_IMMEDIATELY`, and `DATABASE` all can be removed from environment.

```
volumes:
      - ${BENTOV2_FEDERATION_PROD_VOL_DIR}:/federation/data
```

can be removed.

Reply:
done